### DISCUSSION TITLE
Accessing drawer-item and drawer-collection from within a custom interface

### DISCUSSION THREAD

**Discussion by mark-james** [directus/directus#7949] [Archive]
Hi @nickrum,
I've created a custom interface that can visualise O2M data as a chart.  I'd like to leverage the ability to edit or add a new item using the drawer functionality of the O2M list interface. 

Is this possible? 

Or do I need to duplicate these views and package them up as part of the customer interface?

Thanks
Mark

**Comment by nickrum** [comment]
`drawer-item` and `drawer-collection` aren't currently exposed to extensions. You would have to duplicate them for now.
